modules/calibration/data/dev_kit/Config_files/camera.dag

```diff
@@ -0,0 +1,43 @@
+# Define all coms in DAG streaming.
```

Comment:
The dir name "Config_files" makes people a little bit confused since nearly all files under dev_kit is config files :P

Looking at the existing directory names, you may find some thing interesting to follow.